gulpfile.js

```diff
@@ -2,6 +2,7 @@ var fs = require('fs');
 var path = require('path');
 
 var gulp = require('gulp');
+var autoprefixer = require('gulp-autoprefixer');
```

Comment:
There's no need to load it explicitly, because `gulp-load-plugins` will take care of that for you! :smiley: 
So then you only have to refer to it as `plugins.autoprefixer`.